console/console.go

```diff
@@ -198,13 +199,22 @@ func (c *Console) initWeb3(bridge *bridge) error {
 	return err
 }
 
+var defaultAPIs = map[string]string{"eth": "1.0", "net": "1.0", "debug": "1.0"}
```

Comment:
I'd remove `debug` since it's not exposed by default.

Reply:
I'd leave as much as possible -- even if it's not enabled. If we don't know whether it's enabled or not, at least we can leave it up to the user to figure out. Otherwise we'll basically _deny_ access, with no way to enable it, and no way for the end-user to test if it works or not.